src/flamenco/runtime/fd_blockstore.c

```diff
@@ -58,7 +58,10 @@ fd_blockstore_new( void * shmem,
   fd_rwseq_new( &blockstore->lock );
   FD_COMPILER_MFENCE();
 
-  blockstore->off  = 0;
+  blockstore->off         = FD_BLOCKSTORE_ARCHIVE_START;
+  blockstore->fd_size_max = FD_BLOCKSTORE_ARCHIVE_MIN_SIZE;
+  blockstore->lrw_slot    = FD_SLOT_NULL;
+  blockstore->mrw_slot    = FD_SLOT_NULL;
```

Comment:
i would track these in a separate struct `fd_blockstore_archiver` and base the API around that